src/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.csproj

```diff
@@ -12,9 +12,7 @@
 	</ItemGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Data.SqlClient" />
-		<PackageReference Include="Microsoft.SqlServer.DACFx" />
 		<PackageReference Include="Microsoft.SqlServer.SqlManagementObjects" />
-		<PackageReference Include="Microsoft.SqlServer.Management.SmoMetadataProvider" />
```

Comment:
Hm, I don't suppose you know why these were here in the first place?

Reply:
There doesn't seem to be an obvious reason. If dependencies need them, they can import themselves.